test/Tester/LocalhostSiloTests.cs

```diff
@@ -20,13 +21,14 @@ public class LocalhostClusterTests
         [Fact]
         public async Task LocalhostSiloTest()
         {
+            var opts = TestSiloSpecificOptions.Create(new TestClusterOptions(), 1, true);
             var silo = new SiloHostBuilder()
                 .AddMemoryGrainStorage("MemoryStore")
-                .UseLocalhostClustering()
+                .UseLocalhostClustering(opts.SiloPort, opts.GatewayPort)
                 .Build();
 
             var client = new ClientBuilder()
-                .UseLocalhostClustering()
+                .UseLocalhostClustering(opts.GatewayPort)
```

Comment:
The default ports would clash with other tests/runs, so we try to find free ports instead.